GayLesbianBisexualTVCharacters/.htaccess

```diff
@@ -0,0 +1,25 @@
+# Example
+#
+# https://w3id.org/GayLesbianBisexualTVCharacters/1980s/insi redirects to https://home.cc.umanitoba.ca/~wyatt/tv-char1980s.html#insi
+# https://w3id.org/GayLesbianBisexualTVCharacters/road redirects to https://home.cc.umanitoba.ca/~wyatt/tv-characters.html#road
+# https://w3id.org/GayLesbianBisexualTVCharacters/days redirects to https://home.cc.umanitoba.ca/~wyatt/tv-characters.html#days
+#
+# ## Contact
+# This space is administered by:
+#
+# Clair Kronk
+# Clair.Kronk@mountsinai.org
+# GitHub username: Superraptor
+
+# Examples tested using:
+# https://htaccess.madewithlove.com/
+# (22 September 2024)
+
+RewriteEngine On
+RewriteBase /
+
+# Redirect URLs of the form /GayLesbianBisexualTVCharacters/$1
+RewriteRule ^GayLesbianBisexualTVCharacters/([a-z]{1,10})$ https://home.cc.umanitoba.ca/~wyatt/tv-characters.html#$1 [R=301,NE,L]
```

Comment:
RewriteRule patterns are relative to the current directory.  Remove the prefix in all the rules for both these .htaccess files.

Reply:
Sincere apologies-- fixed here: https://github.com/perma-id/w3id.org/pull/4428.